src/shared/services/exporting/favoriteUtils.ts

```diff
@@ -26,6 +26,22 @@ import { Folder } from 'shared/modules/favorites/foldersDuck'
 
 export const CYPHER_FILE_EXTENSION = '.cypher'
 
+export function exportFavoritesAsBigCypherFile(favorites: Favorite[]): void {
+  const fileContent = favorites
```

Comment:
Maybe a matter of taste, but I think a map and join is easier to read than a reduce and template literal
```
const fileContent = favorites
  .map((favorite) => favorite.content)
  .join("\n\n")
  .trim();
```

Reply:
Yeah it's clearer, thanks!